source/ui/EventLoop.ooc

```diff
@@ -0,0 +1,16 @@
+use ooc-base
+import DisplayWindow
+import x11/X11EventLoop
+
+EventLoop: abstract class {
+	init: func
+	processEvents: abstract func (receiver: DisplayWindow)
```

Comment:
For now we can have `receiver` as parameter, but a real event processing system should keep a global list of created widgets and use it to dispatch events.